packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewer.js

```diff
@@ -68,7 +66,6 @@ function SingleImageViewer({
             transform={transform}
           >
             <SVGImageCanvas
-              ref={canvasLayer}
```

Comment:
This ref was originally used for dragging interactions, which caused the Firefox bug.